source/slang/slang-ast-decl.h

```diff
@@ -616,7 +616,7 @@ FIDDLE(abstract)
 class FunctionDeclBase : public CallableDecl
 {
     FIDDLE(...)
-    FIDDLE() Stmt* body = nullptr;
+    Stmt* body = nullptr;
```

Comment:
It's really nice to see that `FIDDLE()` gone; it gives us a lot of reassurance that the functions bodies will not / cannot appear in the serialized AST.